src/App/component/ProjectAvatar.jsx

```diff
@@ -79,9 +90,53 @@ function ProjectAvatar(props) {
     setAddRepoDialogOpen(true)
   }
 
+  const openDeletionAlert = () => {
+    setDeletionAlert(true)
+  }
+
+  const closeDeletionAlert = () => {
+    setDeletionAlert(false)
+  }
+
+  const deleteProject = () => {
+    Axios.post(`http://localhost:9100/pvs-api/project/remove/${props.project.projectId}`, "",
+      {headers: {"Authorization": `${jwtToken}`}})
```

Comment:
Since we can't make sure the `jwt` is in user's localstorage, the `jwt`'s type here will going to be `string | null`, see [this interface](https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L14290) and [doc](https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem) for more details.

That means if the `jwt` is `null`, the generated header will become
```js
// You may try console.log(`${null}`) to see the result

{
  "Authorization": "null"
}
```
which will cause a wrong jwt be sent to our backend.

And there currently seems not have any exception catching implementations for a wrong jwt in our backend.

So suggest proactively determine whether the jwt is null or not.
Maybe use a if-else, or sth tricky like this:

```js
headers: {...(jwt && {"Authorization": jwt})}
```

PS: according to the definition, we don't need to use the template literals here. Just use `{"Authorization": jwt}` would be OK.

Reply:
39daad2